doc/source/groupby.rst

```diff
@@ -980,6 +980,23 @@ To see the order in which each row appears within its group, use the
 
    df.groupby('A').cumcount(ascending=False)  # kwarg only
 
+Enumerate groups
+~~~~~~~~~~~~~~~~
+
+.. versionadded:: 0.19.0
+
+To see the ordering of the groups themselves, you can use the ``enumerate``
+method (compare with ``cumcount``):
+
+.. ipython:: python
+
+   df = pd.DataFrame(list('aaabba'), columns=['A'])
+   df
+
+   df.groupby('A').enumerate()
+
+   df.groupby('A').enumerate(ascending=False)  # kwarg only
```

Comment:
can u add an example (maybe in the examples section of groupby.rst showing where this is useful)